node_launcher/node_set/node_set.py

```diff
@@ -41,6 +41,20 @@ def __init__(self):
         )
         self.lnd_client = LndClient(self.lnd)
 
+        file_name = 'torrc'
+        tor_dir_path = TOR_DIR_PATH[OPERATING_SYSTEM]
+        self.tor_configuration_file_path = os.path.join(tor_dir_path, file_name)
+        log.info(
+            'tor_configuration_file_path',
+            tor_configuration_file_path=self.tor_configuration_file_path
+        )
+        self.tor = Tor(
```

Comment:
@justinmoon here's where that `Tor` class you has asked about would get instantiated (just realised I had included it already on reviewing the codebase today)

Whole thing is still WIP though so don't run this branch or anything as yet 😅